### ISSUE TITLE
Wrong loaded train.xml

### ISSUE THREAD

**Issue by HUN-Droid** [leezer3/OpenBVE#750]
### Description

The game does not load perfectly the train.xml file. I used the CarXmlConvertor app with using separated car.xml files. Each car have the same xml file. The 1st, 3rd and 5th cars are loaded, but cannot load the 2nd, 4th and 6th cars. 
![kép](https://user-images.githubusercontent.com/47685838/156927411-172cbf7d-e8c4-4b75-b538-5e3d31954b38.png)

## Route

Doesn't matter.

## Train

train.xml:
![kép](https://user-images.githubusercontent.com/47685838/156927456-165f32db-3e52-4d4b-8022-075face8c54b.png)
The linked MXA.xml: (some sections are collapsed, because the file is too long)
![kép](https://user-images.githubusercontent.com/47685838/156927494-9604ddbd-154d-4169-b0bd-a1893517b7a9.png)

## Logs

The log.txt:
![kép](https://user-images.githubusercontent.com/47685838/156927560-85e270de-f232-45c2-b0d5-c92389822c40.png)

### Related information 

I have AMD Ryzen 3rd gen processor with more than enough RAM, nVidia graphics
The OS is Windows 10 21H2
OpenBVE Program Version: v1.8.2.3, Stable

**Comment by leezer3** [comment]
That's really odd.
I would assume it's something in the car XML file, but it loads OK on odd numbered cars....

I can't reproduce immediately, but I can at least get the error to look a little less generic, which I've done for the latest nightly build, and it should now be in the log.
(n.b. your related information suggests the current stable version; I'm presuming this is a typo)

**Comment by HUN-Droid** [comment]
No, it's not a typo. I used the latest release. But it doesn't matter, in the today's nightly build have the same problem. (v1.7.859.965-vsts 64bit)

But there is some additional informations about this: the first 3 cars are displayed as the 1st, 3rd and 5th cars. The 4-5-6 defined cars arent loaded. (I tried different named car.xml files, but these had the same content)

The log.txt:
![kép](https://user-images.githubusercontent.com/47685838/156938272-edc6de25-2942-4013-b8aa-27c0df76e639.png)
